Cargo.toml

```diff
@@ -16,10 +16,11 @@ edition = "2018"
 
 [dependencies]
 byteorder = { version = "1", default-features = false }
-ff = "0.7"
-rand = "0.7"
+ff = { version = "0.7", default-features = false }
+rand = { version = "0.7", default-features = false }
 rand_xorshift = "0.2"
 subtle = { version = "2.2.1", default-features = false }
 
-[badges]
-maintenance = { status = "actively-developed" }
```

Comment:
Reason for removing this?

Reply:
Seems I may have been a bit hasty there.

[Crates.io badge support has been removed](https://github.com/rust-lang/crates.io/issues/2436) but apparently there are plans to [eventually add back support for `badges.maintenance` in particular](https://github.com/rust-lang/crates.io/issues/2436#issuecomment-620495491).